### DISCUSSION TITLE
Items per page

### DISCUSSION THREAD

**Discussion by certa-web** [hizzle-co/noptin#940] [Ideas]
![noptin-request](https://github.com/user-attachments/assets/58cd7403-1b68-49bc-a015-45f8a414dd9e)

Hi

We wish an expansion of the page with all subscribers in the backend. 

It is about the display of subscribers. Here we would like to have the opportunity to choose "all". Today, the limit is 100.

Best regards

Brian

**Comment by picocodes** [comment]
This will be added in the next update
